src/playlist-loader.js

```diff
@@ -181,13 +243,13 @@ export const updateMaster = (master, media, unchangedCheck = isPlaylistUnchanged
   // that is referenced by index, and one by URI. The index reference may no longer be
   // necessary.
   for (let i = 0; i < result.playlists.length; i++) {
-    if (result.playlists[i].id === media.id) {
+    if (result.playlists[i].id === newMedia.id) {
       result.playlists[i] = mergedPlaylist;
     }
   }
-  result.playlists[media.id] = mergedPlaylist;
+  result.playlists[newMedia.id] = mergedPlaylist;
   // URI reference added for backwards compatibility
-  result.playlists[media.uri] = mergedPlaylist;
+  result.playlists[newMedia.uri] = mergedPlaylist;
 
   // update media group playlist references.
   forEachMediaGroup(master, (properties, mediaType, groupKey, labelKey) => {
```

Comment:
chore: in this fold, on linees 260 and 261 in the loops, we still refer to `media` but presumably, it should refer to `newMedia`? I can't comment on the direct line because of github. this is also why the build is failing on this PR.